src/views/unauthenticated/About.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+
+export function About() {
+	return (
+		<div>
+			<h1>ABOUT APPLICATION NAME</h1>
+			<section>
+				<h2>How works</h2>
+				<ul>
+					<li>
+						Create a list for different stores or different grouping of items.
+					</li>
+					<li>Select a list that you&apos;d like to add items on.</li>
+					<li>
+						Open the list manager, to start adding items and choosing when you
+						will need to restock next.
+					</li>
+					<li>
+						Now that your new list has items you can start checking off items as
+						you shop!
+						<ul>
+							<li>
+								Each time an item is marked purchase the application evaluates
+								your shopping habits! Supporting you by adjusting your next
+								purchase predictions base on when your previous shopping
+								history!
+							</li>
+						</ul>
+					</li>
+					<li>
+						If someone else needs to be let into to the shopping time you can
+						easily share specific lists with with in the list manager so they
+						can see and mark items as purchased too!
+					</li>
+				</ul>
+			</section>
+			<section>
+				<h2>Creators</h2>
+				<div>
+					<p>Maha Ahmed</p>
+				</div>
```

Comment:
User [icons8](https://icons8.com/icons/set/linkedin) for now, not sure if a different set of free icons would like to be used.

Reply:
Try canva! https://www.canva.com/learn/free-icons-download/ I'll see if I can find what I used in the past for some of my food icons.

Reply:
Yeah I think this is linked somewhere and I honestly just clicked the first one there since we hadn't used many icons yet I didn't want to just assume but needed something nice for the page 😅